examples/object-value-from-class-definition/object_value_from_class_definition.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/io;
+
+class Engineer {
+    string name;
+
+    function init(string name = "Null") {
+        self.name = name;
+    }
+
+    function getName() returns string {
+        return self.name;
+    }
+}
+
+public function main() {
+    // Explicit new expression.
+    // Object is initialized using the default value in the `init` method.
+    Engineer engineer1 = new Engineer();
+    io:println(engineer1.getName());
+
+    // Object is initialized using the given argument to new expression.
+    Engineer engineer2 = new Engineer("Alice");
+    io:println(engineer2.getName());
+
+    // Object is initialized using the given argument to new expression.
```

Comment:
```suggestion
    // Object is initialized to a new expression using the given argument.
```
Check if the technical meaning is affected by the suggestions.

Reply:
Updated e92bec5125fa19f0265d690fe4ae313621a9f8b0